tests/integration/targets/azure_rm_devtestlab/tasks/main.yml

```diff
@@ -0,0 +1,91 @@
+- name: Prepare random number
+  set_fact:
+    rpfx: "{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
```

Comment:
i'm curious on why you not just  direct set name  variable in set_fact, but appending rpfx at  each of test, which is error-prone, because easy to forget or typo...